pkg/pgmodel/migrate.go

```diff
@@ -375,7 +373,7 @@ func orderFilesNaturally(entries []os.FileInfo) []string {
 	return pp.getNames()
 }
 
-func replaceSchemaNames(r io.ReadCloser) (string, error) {
+func readSchemas(r io.ReadCloser) (string, error) {
```

Comment:
`readSchemas` might not be the best name here.
```suggestion
func readMigrationFile(r io.ReadCloser) (string, error) {
```

Reply:
Done